src/studio/src/designer/backend/Services/Implementation/SourceControlSI.cs

```diff
@@ -236,7 +237,14 @@ public async Task Push(string org, string repository)
                     repo.Network.Remotes.Update("origin", r => r.Url = remoteUrl);
                 }
 
-                PushOptions options = new PushOptions();
+                PushOptions options = new PushOptions 
+                {
+                    OnPushStatusError = pushError => 
+                    {
+                        _logger.Log(Microsoft.Extensions.Logging.LogLevel.Error, "Push error: {0}", pushError.Message);
```

Comment:
could use the built in methods LogInformation / LogError to simplify these logs, but no biggie

Reply:
Fixed.